sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -117,7 +117,9 @@ page at http://checkstyle.sourceforge.net/config.html -->
     </module>
 
     <!-- Check that paragraph tags are used correctly in Javadoc. -->
-    <!-- <module name="JavadocParagraph"/> -->
+    <!--
+    TODO(https://issues.apache.org/jira/browse/BEAM-4670): re-enable after fixing after spotless
```

Comment:
I believe your earlier commit [removes the checkstyle check](https://github.com/apache/beam/pull/5797/commits/80fc678959c23de0ed88aa6f69d33d99756bf623#diff-86bc58f0541253b6301474bf7310a5e8L400 which previously banned this type of TODO.

The check banned having content tied to a TODO, i.e. `TODO(kenn): foobar`. The check makes sense for usernames, but seems totally backwards for linking bugs.

I'm happy keeping this format around. I'd also suggest shortening to simply `TODO(BEAM-4670): ...`.

Reply:
I have a slight difference of opinion in that it drives me bananas when people refer to tickets without the full URL for me to mindlessly copy/paste :-). But I get what you mean - it is more readable to just have the identifier rather than all the cruft.

Reply:
Incidentally, I hit this checkstyle issue earlier and intended to remove that check anyhow. We can `@Ignore(<url>)` but we can't `TODO(<url>)`. It made little sense. It was a carryover from before Apache Beam.